### PR TITLE
Document the environment variables used in bindings_cfg()

### PR DESCRIPTION
**Pull request by maggu**
I think these ought to be documented. It would greatly have simplified getting the tests running for us.

### CODE REVIEW

**Review by mdellweg — DISMISSED** [review]

**Comment by maggu** [comment]
@mdellweg  Thank for the quick approval. Sorry for breaking it by adding a towncrier entry. 😃

File: CHANGES/5807.doc [hunk 1]
```diff
@@ -0,0 +1 @@
+Document the environment variables used in bindings_cfg()
```
Comment (mdellweg) [inline CHANGES/5807.doc:1]: Let's be more specific here (Sprinkling a "test" there should help the audience to filter it):
```suggestion
Document the environment variables used to hydrate the test fixture `bindings_cfg`.
```

Comment (maggu) [reply CHANGES/5807.doc:1]: Sounds good. Thanks. Might as well adjust the commit message then as well. Why "to hydrate" though? Maybe just use "by" or "in" instead?

Comment (mdellweg) [reply CHANGES/5807.doc:1]: That'd be fine too. Important is that a reader only be _using_ Pulp knows this is about tests and does not concern them.

Comment (maggu) [reply CHANGES/5807.doc]: 👍 Updated now.

**Review by mdellweg — APPROVED** [review]

**Pull request merged**